lib/util/cache/repository/types.ts

```diff
@@ -40,4 +40,6 @@ export interface Cache {
   init?: RepoInitConfig;
   scan?: Record<string, BaseBranchCache>;
   lastPlatformAutomergeFailure?: string;
+  baseBranchSha?: string;
```

Comment:
I don't think this is the correct place. There's no single `baseBranchSha` in a repo. There's not only multiple base branch names.. but we will have multiple PRs merged